src/docs/product/integrations/integration-platform/ui-components/issue-link.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Issue Link
+sidebar_order: 1
+description: "Learn about the issue link UI component."
+---
+
+With an issue linking UI component, you can connect Sentry issues with a task in your project management system. This functionality provides a way to use any project management tool you use or develop.
+
+The issue link component displays with the text "Link &lt;Service&gt; Issue" in the **Issue Details** sidebar:
+
+![Issue linking sidebar showing Shortcut icon.](issue-sidebar-icons.png)
+
+Clicking the link opens a modal allowing the user to create or link the issue to a task in the external service:
+![Sentry modal that creates a Shortcut Story that's linked to a Sentry issue.](../../issue-tracking/shortcut/shortcut_create_story_with_issue.png)
+
+When an external issue is created or linked in [sentry.io](https://sentry.io), a display name is shown that links back to the service where it was either created or linked. The display name is composed of two pieces: the `project` and the `identifier`. A hash (#) connects each piece. Here's an example of what it looks like in [sentry.io](https://sentry.io):
+
+![Sentry's sidebar button illustrating that an issue is linked.](../../issue-tracking/shortcut/shortcut_story_in_linked_issues_82.png)
+
+## Schema
+
+```json {filename:schema.json}
+{
+  "elements": [
+    {
+      "type": "issue-link",
+      "link": {
+        "uri": <String>,
+        "required_fields": <Array<FormField>>,
+        "optional_fields": <Array<FormField>>
+      },
+      "create": {
+        // Same as "link" schema
+      }
+    }
+  ]
+}
+```
+
+### Attributes
```

Comment:
This can be an h2 to match the other UI component pages, but I also get if you want to keep it h3, since there is already another 'Attributes' heading on the page that makes sense to be a h3